src/main/kotlin/com/cognifide/gradle/aem/bundle/BundleJar.kt

```diff
@@ -215,6 +218,22 @@ val jar: Jar
         if (!hasAttribute(Bundle.ATTRIBUTE_SLING_MODEL_PACKAGES) && !javaPackage.isNullOrBlank()) {
             slingModelPackages = javaPackage
         }
+
+        if (!hasAttribute(Bundle.ATTRIBUTE_ACTIVATOR) && !javaPackage.isNullOrBlank()) {
+            findActivator(javaPackage!!)?.let { activator = it }
```

Comment:
new feature, automatically added manifest entry if activator class will be detected in proper place